problem_builder/public/css/questionnaire.css

```diff
@@ -90,13 +90,15 @@
     display: none;
     cursor: pointer;
     position: absolute;
-    top: 5px;
-    right: 3px;
-    width: 18px;
-    height: 21px;
-    font-family: FontAwesome;
-    font-style: normal;
-    text-align: center;
```

Comment:
Do we actually need to remove this? Wouldn't it be sufficient to just add `font-size: 0;` or am I missing something??

Reply:
@msaqib52 Please get this done by shahid to get this PR merged asap

Reply:
@Agrendalath Ah! I have removed all the css changed for this fix. Now we have left only 2 files with 2 changes. Please go ahead and merge after review

Reply:
Thanks for the changes, @murad-hubib. I'll review this tomorrow.